website/client/components/chat/chatCard.vue

```diff
@@ -17,25 +17,28 @@ div
       span.mr-1(v-if="msg.username") •
       span(v-b-tooltip="", :title="msg.timestamp | date") {{ msg.timestamp | timeAgo }}
     .text(v-html='atHighlight(parseMarkdown(msg.text))')
+    .reported(v-if="isMessageReported && inbox")
+      span(v-once) {{ $t('reportedMessage')}}
+      br
+      span(v-once) {{ $t('canDeleteNow') }}
```

Comment:
Perhaps the text only appears if you've flagged a group message after recently having the private message inbox open?

Reply:
will try to reproduce this tmrw

Reply:
This is still happening although only for moderators/staff.

Reply:
I set my account to `admin: true` and `level: 8` and I still don't get the `reportedMessage`/`canDeleteNow` messages:

Reported messages in party / groups / tavern / guilds and private messages and I still not getting em.

Do you have an example group / database you could send me to test it?

Reply:
https://habitrpg-delta.herokuapp.com/groups/tavern

![image](https://user-images.githubusercontent.com/1495809/50735085-9b7f1480-11f4-11e9-9d3d-94bdb8171521.png)

```
{
    "_id": "9c0b5eb8-6c86-48a8-b2c0-fcb300dbdbf2",
    "flagCount": 5,
    "flags": {
        "d904bd62-da08-416b-a816-ba797c9ee265": true
    },
    "id": "9c0b5eb8-6c86-48a8-b2c0-fcb300dbdbf2",
    "text": "Hi",
    "timestamp": "2019-01-06T10:28:50.214Z",
    "likes": {},
    "client": "ios",
    "uuid": "xxxxxx,
    "contributor": {},
    "backer": {},
    "user": "xxxxx",
    "username": "xxxxx",
    "groupId": "00000000-0000-4000-A000-000000000000",
    "__v": 0
}
```

I've redacted the user's names and ID since it's an innocent message.  The post was made about 25 minutes ago and I flagged it about 20 minutes ago.

Are you doing your testing on delta or in your local install? Maybe there's code locally that hasn't been pushed to GitHub yet? Or perhaps there's code in this PR that's not on delta yet, although I thought Sabe deployed it fairly recently (at the very least in the past month unless I'm badly misremembering).

If you like, put this specific problem on hold until after the next deployment to delta. We can at least test the `_DISPLAY_NAME` change now. I'll report back here after that.

@SabreCat when you have time, can you deploy this PR to delta again?

Reply:
It's still happening on delta after the latest delploy.

Reply:
I refactored the properties to add a default value, what I guess is that the binding of `undefined to :inbox="inbox"` may caused the inner checks of `inbox` being true which shouldn't be, if the flagged text still exists after that I'm out of ideas, since I can't reproduce it locally